website/docs/r/postgresql_flexible_server_database.html.markdown

```diff
@@ -0,0 +1,77 @@
+---
+subcategory: "Database"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_postgresql_flexible_server_database"
+description: |-
+  Manages a PostgreSQL Flexible Server Database.
+---
+
+# azurerm_postgresql_flexible_server_database
+
+Manages a PostgreSQL Flexible Server Database.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
```

Comment:
fwiw we don't need the provider block in examples (since users may have different features enabled)

```suggestion
```

Reply:
removed